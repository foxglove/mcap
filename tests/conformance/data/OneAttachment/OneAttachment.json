{
  "records": [
    {"type": "Header", "fields": [["library", ""], ["profile", ""]]},
    {
      "type": "Attachment",
      "fields": [
        ["create_time", "1"],
        ["data", ["1", "2", "3"]],
        ["log_time", "2"],
        ["media_type", "application/octet-stream"],
        ["name", "myFile"]
      ]
    },
    {"type": "DataEnd", "fields": [["data_section_crc", "3051995293"]]},
    {
      "type": "Footer",
      "fields": [
        ["summary_crc", "1875167664"],
        ["summary_offset_start", "0"],
        ["summary_start", "0"]
      ]
    }
  ],
  "meta": {"variant": {"features": []}}
}
