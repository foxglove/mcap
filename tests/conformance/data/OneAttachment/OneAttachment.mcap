version https://git-lfs.github.com/spec/v1
oid sha256:816f7ef60af488cccbb67fbe7f2e3de2abf788793ba548993254140dd8a95acc
size 153
