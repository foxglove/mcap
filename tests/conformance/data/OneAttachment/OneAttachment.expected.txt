Header library="" profile=""
Attachment content_type=application/octet-stream created_at=1 data=<010203> log_time=2 name=myFile
AttachmentIndex content_type=application/octet-stream data_size=3 length=78 log_time=2 name=myFile offset=25
Footer summary_offset_start=0 summary_start=116
